src/webgl/light.js

```diff
@@ -113,27 +113,40 @@ p5.prototype.ambientLight = function(v1, v2, v3, a) {
  * @example
  * <div>
  * <code>
+ * let setSpecularColor = true;
```

Comment:
`setRedSpecular`? In simple examples it feels good to see as much info as possible about the impact of a boolean in its name.